test/functional/wallet_importmulti.py

```diff
@@ -416,15 +418,16 @@ def run_test(self):
                           timestamp=timestamp)
 
         # Address + Public key + !Internal + Wrong pubkey
-        self.log.info("Should not import an address with the wrong public key as non-solvable")
+        self.log.info("Should import an address with the wrong public key as non-solvable")
         key = self.get_key()
         address = key.p2pkh_addr
+        pkh = hash160(hex_str_to_bytes(key.pubkey))
         wrong_key = self.get_key().pubkey
         self.test_importmulti({"scriptPubKey": {"address": address},
                                "timestamp": "now",
                                "pubkeys": [wrong_key]},
                               True,
-                              warnings=["Importing as non-solvable: some required keys are missing. If this is intentional, don't provide any keys, pubkeys, witnessscript, or redeemscript.", "Some private keys are missing, outputs will be considered watchonly. If this is intentional, specify the watchonly flag."])
+                              warnings=["Importing as non-solvable: some required keys are missing: " + bytes_to_hex_str(pkh) + ". If this is intentional, don't provide any keys, pubkeys, witnessscript, or redeemscript.", "Some private keys are missing, outputs will be considered watchonly. If this is intentional, specify the watchonly flag."])
```

Comment:
nit (this isn't introduced here, but since this PR is for nit-clearing...): this line is very long. Consider:

- building a warnings variable above and passing it to this function
- adding constants for the common error strings to avoid repetition
- using `.format()` string formating instead of string concatenation.